site/docs/BestPractice-ServingOverHTTP.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Serving over HTTP
+layout: ../_core/DocsLayout
+category: Best Practices
+permalink: /learn/serving-over-http/
+next: /learn/authorization/
+---
+
+HTTP is a great choice for client-server communication when using GraphQL because of its ubiquity. Here are some guidelines for setting up a GraphQL server to operate over HTTP.
```

Comment:
Could be a bit more specific and say it's the best / most common choice. It seems unlikely you'd want something other than HTTP, unless you want something more realtimey.